### PR TITLE
CRW-1093 copy downstream Dockerfile to...

### PR DESCRIPTION
**Pull request by nickboldt**
CRW-1093 copy downstream Dockerfile to rhel.Dockerfile upstream so we can better sync che server w/ crw server

Change-Id: Id76c6c38c4e17708e577de8abc5acde4b8376edf
Signed-off-by: nickboldt <nboldt@redhat.com>

### CODE REVIEW

**Review by amisevsk — APPROVED** [review]

**Comment by benoitf** [comment]
how it can be merged in a backport branch before master approval of code owners ?

**Pull request merged**
